lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -3,6 +3,190 @@
  * Licensed under the MIT License.
  */
 
+import * as msalAuth from "msal-common";
+import { Configuration, buildConfiguration } from "./Configuration";
+
+/**
+ * Interface to handle iFrame generation, Popup Window creation and redirect handling
+ */
+declare global {
```

Comment:
Why should this be in a public interface? Can we have this in an internal file?

Reply:
This was copied from UAA, and I'm assuming it's because this is the general entry point for the library. I will do some research around how defining globals should work.